### PR TITLE
Add Request a Quote variation to Form block

### PR DESCRIPTION
**Pull request by jlemay-godaddy**
### Description
Adding the "Request a Quote" variation to the "Form" block.

### Screenshots
image
image

### Types of changes
New variation

### How has this been tested?
Manually tested

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Comment by jonathanbardo** [comment]
@jlemay-godaddy can you rebase master so the phpunit test are runned correctly?

**Comment by olafleur-godaddy** [comment]
Great job!

I would maybe add a few Jest tests at least, so that the guy who raised our coverage near 100% won't be mad at you for dropping it ;)

**Comment by olafleur-godaddy** [comment]
Never mind, it's a variation of the Form block so it's already well tested. Maybe a Cypress test to test the variation, then ;)

File: package.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
 		"zh_TW": "繁體中文"
 	},
 	"dependencies": {
-		"@godaddy-wordpress/coblocks-icons": "1.8.1"
```
Comment (olafleur-godaddy) [inline package.json:76]: You can update to 1.10.0 if you want ;)

**Review by olafleur-godaddy — APPROVED** [review]

**Pull request merged**